src/modules/Verifications/components/ProofRequestModal/index.tsx

```diff
@@ -40,7 +40,15 @@ export const ProofRequestModal: React.FC<ProofRequestProps> = ({
             {!shortLink && loading && <CircularProgress size="4rem" sx={{ color: variables.color_primary_600 }} />}
             {!shortLink && loading === undefined && <img src={qrCode} alt="QR Code" height={240} width={240} />}
           </div>
-          <Button variant="contained" color="primary" fullWidth>
+          <Button
+            variant="contained"
+            color="primary"
+            fullWidth
+            onClick={() => {
+              const newTab = window.open(shortLink, '_blank');
```

Comment:
couldn't you use `to` and `target` props?

Reply:
It is not final yet. We just focused on functionality. I am not sure about the onClick function. Ehsan is going to replace it later